src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -277,7 +312,12 @@ private static void validateTableAccess(Client engineClient, JSONObject mapping,
                     }
                 }
             }
-        } catch (DataClientException e) {
+            if (hasAccess && shouldCheckStreaming && !isStreamingPolicyEnabled(MAPPING_TABLE, table, engineClient, database)) {
```

Comment:
Do you want to fall back to queued ingestion ?

Reply:
i don't